### PR TITLE
Rdiscrowd 1536 2

### PR DESCRIPTION
**Pull request by mariarivera30**
ThIs PR is not complete. Test is missing. Before doing test I . want to get feedback or question about it.

This PR is to standardize how we store gold_ans for public and priv . 

For public Gig gold_answers ={...all answers}
For priv GIG gold_answers ={gold_ans_url: "File that has gold_answers as a JSON"}

### CODE REVIEW

File: pybossa/importers/importer.py [hunk 1]
```diff
@@ -124,34 +125,11 @@ def register_youtube_importer(self, youtube_params):
...
-        if not (private_fields or private_gold_answers):
+        if not private_fields:
             return
-
-        encryption = current_app.config.get('ENABLE_ENCRYPTION', False)
-        bucket = current_app.config.get("S3_REQUEST_BUCKET")
-        task_hash = hashlib.md5(json.dumps(task)).hexdigest()
-        path = "{}/{}".format(project_id, task_hash)
-        s3_conn_type = current_app.config.get('S3_CONN_TYPE')
         if private_fields:
```
Comment (dchhabda) [inline pybossa/importers/importer.py]: this condition can be removed

File: pybossa/task_creator_helper.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf8 -*-
+# This file is part of PYBOSSA.
+#
+# Copyright (C) 2015 Scifabric LTD.
```
Comment (dchhabda) [inline pybossa/task_creator_helper.py]: 2019 here

File: pybossa/importers/importer.py (hunk 1 above)
Comment (mariarivera30) [reply pybossa/importers/importer.py]: Will do.

File: pybossa/task_creator_helper.py (hunk 2 above)
Comment (mariarivera30) [reply pybossa/task_creator_helper.py]: failed copy paste

**Pull request merged**